packages/e2e/alonzo-private-network/docker-compose.yml

```diff
@@ -0,0 +1,134 @@
+version: "3.3"
+
+services:
+  private-testnet:
```

Comment:
```suggestion
  local-test-network:
```

Reply:
fixed in https://github.com/input-output-hk/cardano-js-sdk/commit/d963d0afe05db27b88d1451925c08e851f749877